@types/automerge/index.d.ts

```diff
@@ -0,0 +1,253 @@
+declare module 'automerge' {
+  function applyChanges<T>(doc: T, changes: Change[]): T
+  function canRedo<T>(doc: T): boolean
+  function canUndo<T>(doc: T): boolean
+  function change<T>(doc: T, message: string, callback: ChangeFn<T>): T
```

Comment:
This should be `message: string | undefined`.

```
let message = ifCondition ? 'Yay' : undefined;
const newDoc = Automerge.change(doc, message, (change: any) => {
              ...
});
```

Reply:
✅ fixed in 30cf6e